packages/c2pa-wc/src/components/Indicator/Indicator.ts

```diff
@@ -52,6 +52,6 @@ export class Indicator extends LitElement {
   }
 
   render() {
-    return html`<cai-icon-info class="icon" />`;
+    return html`<cai-icon-info class="icon" /> `;
```

Comment:
Please remove trailing space.
```suggestion
    return html`<cai-icon-info class="icon" />`;
```